sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,22 +1,20 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
-
-### Features Added
+## 1.0.0-beta.5 (2024-11-18)
 
 ### Breaking Changes
 
 - Renamed `tables_clients.hpp` to `table_client.hpp` and split `TableServiceClient` into its own file, `table_service_client.hpp`. 
 - Removed the `TablesAudience` field from `TableClientOptions` since it is not required.
 - Removed ServiceVersion type and changed the ApiVersion field within `TableClientOptions` to be std::string.
 - Removed the `TableServiceClient` constructor that only accepts one defaulted options parameter.
+- Updated samples to reflect the changes in the client.

```

Comment:
This is under "Breaking Changes", but is it a breaking change?
Move it under "Other Changes"?